canonical-web-and-design/ubuntu.com
# Replaced the g-recatcha with two honeypot fields

## Done
- Replaced the g-recatcha with two honeypot fields

Do no merge until this one merges: https://github.com/canonical-web-and-design/ubuntu.com/pull/9590/

## QA
- Go to /engage/multi-cloud-business-guide and fill in the form and see if work
- Go back to /engage/multi-cloud-business-guide and inspect the form and add a value to the website or name hidden fields then submit and check it errors
- Go to /core/contact-us and check that works as before
- Go to /internet-of-things#get-in-touch and check that works as before.

## Issue / Card
Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/9753
